### ISSUE TITLE
eCRNOW: use attribute is not populated in the patient name element

### ISSUE THREAD

**Issue by AnanthaKrishnaV** [drajer-health/eCRNow#465]
eCRNOW: use attribute is not populated in the patient name element.

/ClinicalDocument/recordTarget/patientRole/patient/name
In CdaGeneratorConstants class below method is comparing string with case senstive, so it fails to add the "use" attribute to name element. The caller of this method used getUse().toString() instead it should use getUse().toCode().
```
 public static String getCodeForNameUse(String val) {

    if (!StringUtils.isEmpty(val)) {

      if (val.contentEquals("official") || val.contentEquals("usual")) {
        return "L";
      } else {
        return null;
      }

    } else {
      return null;
    }

    // Unable to translate Anonymous, Temp and Old which is present in FHIR but not in CDA

  }
```

**Comment by nbashyam** [comment]
@AnanthaKrishnaV  - Assuming you will send a PR with the above fix.

**Comment by AnanthaKrishnaV** [comment]
Addressed with PR #470

**Comment by AnanthaKrishnaV** [comment]
> @AnanthaKrishnaV - Assuming you will send a PR with the above fix.

Yes PR #470
